src/components/section-display.js

```diff
@@ -48,19 +48,21 @@ function SectionDisplay({
 
   const isHardDeleteEnabled = canHardDelete && !section.deletedOn;
 
+  const [isEditing, setIsEditing] = useState(false);
+  const [isInvalid, setIsInvalid] = useState(false);
+  const [isHelpModelVisible, setIsHelpModalVisible] = useState(false);
```

Comment:
I know this was there before, but it'd be a good opportunity to rename the model to modal :laughing:

Reply:
Yes, we don't want these species mixed! 😱 Thanks!